metasrv/src/meta_service/raftmeta.rs

```diff
@@ -809,4 +821,8 @@ impl MetaNode {
         let res: Result<ForwardResponse, MetaError> = raft_mes.into();
         res
     }
+
+    pub fn create_watcher_stream(&self, request: Request<WatchRequest>, tx: WatcherStreamSender) {
+        self.watcher.create_watcher_stream(request, tx)
+    }
```

Comment:
Except for the service endpoint layer, these method does not need to know about tonic request.

A `WatchRequest` is quite enough here.

```suggestion
    pub fn create_watcher_stream(&self, request: WatchRequest, tx: WatcherStreamSender) {
        self.watcher.create_watcher_stream(request, tx)
    }
```